docs/operate/docker-quickstart.mdx

```diff
@@ -147,13 +177,15 @@ Bring the containers up:
 Remember to fund your validator address so the validator can announce.
 
 ```bash
-docker-compose --env-file .env.ethereum up -d
-docker-compose --env-file .env.optimism up -d
-docker-compose --env-file .env.base up -d
+docker compose --env-file .env.ethereum up validator1 -d
+docker compose --env-file .env.optimism up validator2 -d
+docker compose --env-file .env.base up validator3 -d
 ```
 
 Ensure there are no errors:
 
 ```bash
-docker logs -f ethereum-validator
+docker logs -f validator1
+docker logs -f validator2
+docker logs -f validator3
```

Comment:
would prefer more descriptive service names e.g. `<chainname>-validator`

Reply:
I updated the doc accordingly